homeassistant/components/touchline/climate.py

```diff
@@ -1,4 +1,9 @@
-"""Platform for Roth Touchline heat pump controller."""
+"""
+Platform for Roth Touchline floor heating controller.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/touchline/
```

Comment:
We don't add the url to the docstring anymore. Just keep the very first line. Ie please revert the change.